auto-reconfiguration.sh

```diff
@@ -9,6 +9,7 @@ VERSION=$(cat auto-reconfiguration-archives/version)
 INDEX_PATH="/auto-reconfiguration/index.yml"
 UPLOAD_PATH="/auto-reconfiguration/auto-reconfiguration-$VERSION.jar"
 
+rm -f auto-reconfiguration-archives/auto-reconfiguration-*sources.jar
```

Comment:
I didn't see any reason to upload them as they aren't being used further down the pipeline. Would it to be sure we have a record of these?